src/commands/lambda/instrument.ts

```diff
@@ -55,6 +61,16 @@ export class InstrumentCommand extends Command {
       return 1
     }
 
+    if (this.sourceCodeIntegration) {
+      if (!process.env.DATADOG_API_KEY) {
```

Comment:
I think we want to support `DATADOG_KMS_API_KEY` as well here

Reply:
Fixed, let me know if it looks good!